### ISSUE TITLE
Making things digitally

### ISSUE THREAD

**Issue by pedrottimark** [gaearon/ama#18]
When did you start making things digitally?
When did you know it had become an important part of your life?

**Comment by pedrottimark** [comment]
First question has already been asked, my bad.

**Comment by gaearon** [comment]
I was interested in how things work for as long as I remember myself. I used to disassemble any tech my family wanted to throw away, like clocks, telephones, etc.

When I was 8 or 9 years old, I read books on circuit engineering. I drew circuits of clocks and similar stuff, and I made a shopping list: resistors, logic elements, soldering board, etc. However after I bought all of that, I was too lazy to actually do the soldering, and I never proceeded with that hobby.

It's only after I got a computer and found out about programming (#2) that I started actually making something. Again, for a long time, I didn't actually _create_ anything so I was just helping people on forums. My first “real” project was [tagmonkey](https://code.google.com/p/tagmonkey)—a now-defunct iTunes extension to auto-tag music library including searching for artwork in Google Images and lyrics in LyricWiki.

However I only really got into open source after releasing React Hot Loader which received tremendous attention from the community, and motivated me to release stuff in the open.
